pkg/registry/batch/job/strategy_test.go

```diff
@@ -749,6 +749,7 @@ func TestJobStrategy_WarningsOnUpdate(t *testing.T) {
 					Generation:      0,
 				},
 				Spec: batch.JobSpec{
+					CompletionMode: completionModePtr(batch.NonIndexedCompletion),
```

Comment:
nit: why adding this line? It seems unrelated.

Reply:
I see, please do in the existing places. Adding in the new tests is good.

Reply:
removed.